pkg/api/api.go

```diff
@@ -78,6 +84,7 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 func (cfg *Config) RegisterFlagsWithPrefix(prefix string, f *flag.FlagSet) {
 	f.StringVar(&cfg.AlertmanagerHTTPPrefix, prefix+"http.alertmanager-http-prefix", "/alertmanager", "HTTP URL path under which the Alertmanager ui and api will be served.")
 	f.StringVar(&cfg.PrometheusHTTPPrefix, prefix+"http.prometheus-http-prefix", "/prometheus", "HTTP URL path under which the Prometheus api will be served.")
+	f.StringVar(&cfg.corsRegexString, prefix+"server.cors-origin", ".*", "Access-Control-Allow-Origin response header. Regex anchors are not added by default.")
```

Comment:
I don't quite understand `Regex anchors are not added by default.`. Can we reuse the same description as Prometehus https://github.com/prometheus/prometheus/blob/main/cmd/prometheus/main.go#L296

Reply:
updated